go/teams/implicit_test.go

```diff
@@ -539,22 +539,29 @@ func TestReAddMemberWithSameUV(t *testing.T) {
 	jun := fus[2] // pukless user
 	hal := fus[3] // pukless user (eldest=0)
 
-	kbtest.ResetAccount(*tcs[3], fus[3])
+	tcAnn := tcs[0] // crypto user
```

Comment:
please check that my changes to this existing test are good. 

mostly trying to improve readability. and i was confused about a couple calls to ReAddMemberAfterReset (when i think they should be reAddMemberAfterResetInner), please check those in particular.

Reply:
is it because of `ReAddMemberAfterReset` silencing one of the errors?

Reply:
yeah

Reply:
no clue, i looked at some of the tests and it is kind of weird. cc @mlsteele do you still remember that change?

Reply:
It looks like `kbtest.ResetAccount(*tcs[3], fus[3])` may have been removed because it looks like it doesn't do anything here but it does. `setupNTestsWithPukless` signs up hal and provisions a device, after which hal is at eldestSeqno=1. `kbtest.ResetAccount(*tcs[3], fus[3])` resets hal so that they're at eldestSeqno=0. Re-adding a user who is at eldestSeqno=0 is a state worth testing imo.

I just pushed an update to `script eldest` so that it prints the `current_eldest` seqno of a user.
```
$ kbu script eldest team_79e15fdce4
{"current_eldest":1,"seqnos":[1]}
```

As for testing `reAddMemberAfterResetInner` vs `ReAddMemberAfterReset` I'm not quite sure what change you mean. But it seems nice to test both, which this looks like it's doing. If I had to pick one it'd be `ReAddMemberAfterReset` since that's what the gui caller experiences.

supressed -> suppressed

Reply:
line `kbtest.ResetAccount(*tcs[3], fus[3])` was replaced with line 546 `kbtest.ResetAccount(*tcHal, hal) // reset hal`

Reply:
> supressed -> suppressed
i'll fix this typo

Reply:
> was replaced

ha, sorry, great.